src/program.js

```diff
@@ -103,7 +103,9 @@ export class Program {
       await runCommand(argv);
     } catch (error) {
       const prefix = cmd ? `${cmd}: ` : '';
-      log.error(`\n${prefix}${error.stack}\n`);
+      if (!error instanceof UsageError) {
+        log.error(`\n${prefix}${error.stack}\n`);
```

Comment:
you also need to log this line if `argv.verbose` is true, regardless of whether it's a `UsageError` or not.